zifter/package.yaml

```diff
@@ -52,9 +52,9 @@ tests:
     - zifter
     - QuickCheck >=2.9 && <2.11
     - colour >=2.3 && <2.4
-    - genvalidity >=0.3 && <0.5
-    - genvalidity-hspec >=0.3 && <0.6
-    - genvalidity-path >=0.1 && <0.3
+    - genvalidity >=0.4 && <0.5
+    - genvalidity-hspec >=0.5 && <0.6
+    - genvalidity-path >=0.2 && <0.3
```

Comment:
It's better to upgrade the upper bound if necessary, but maintain backward compatibility.